_includes/feedback.html

```diff
@@ -0,0 +1,23 @@
+<div class="alert alert-info alert-dismissible fade show" role="alert">
```

Comment:
can you make this page name more future proof in case we ask for other feedback in the future? how about "install-feedback.html"?